src/components/calcite-slider/calcite-slider.stories.ts

```diff
@@ -13,43 +13,75 @@ export default {
 };
 
 export const SingleValue = (): string => html`
-  <calcite-slider
-    min="${number("min", 0)}"
-    max="${number("max", 100)}"
-    value="${number("value", 50)}"
-    step="${number("step", 1)}"
-    label="${text("label", "Temperature")}"
-    ${boolean("disabled", false)}
-    ${boolean("label-handles", false)}
-    ${boolean("label-ticks", false)}
-    ticks="${number("ticks", 0)}"
-    page-step="${number("page-step", 5)}"
-    ${boolean("precise", false)}
-    ${boolean("snap", true)}
-  ></calcite-slider>
+  <div style="width: 250px;">
```

Comment:
Just curious, does the story look crazy w/o specifying a width?

Reply:
Yes all of them do. Slider relies on a width being set on its parent container or a min-width style set on it (existing behavior). Without these styles, slider has 0 width and all you see is a thumb handle.

I considered adding a min-width: 150px; or similar to its :host styles to fix this issue, but instead you can see I just moved the Storybook styles from preview-head.html into a container div.

I'm going to revert these changes so slider retains its existing layout behavior.